problems/[230612] Implement Queue using Stacks.js

```diff
@@ -0,0 +1,60 @@
+const MyQueue = function () {
+  this.stack1 = [];
+  this.stack2 = [];
+};
+
+/**
+ * @param {number} x
+ * @return {void}
+ */
+MyQueue.prototype.push = function (x) {
+  this.stack1.push(x);
+};
+
+/**
+ * leetcode problem link: https://leetcode.com/problems/implement-queue-using-stacks
+ *
+ * @return {number}
+ */
+MyQueue.prototype.pop = function () {
+  if (this.stack2.length) {
+    return this.stack2.pop();
+  }
+
+  while (this.stack1.length) {
+    this.stack2.push(this.stack1.pop());
+  }
+
+  return this.stack2.pop();
+};
```

Comment:
stack2에는 stack1의 reverse된 데이터가 들어가는 거군요..! 

```suggestion
  if (!this.stack2.length) {
    while (this.stack1.length) {
      this.stack2.push(this.stack1.pop());
    }
  }

  return this.stack2.pop();
};
```

민주님이 하신 early return 방식도 좋고.. 이 방식도 나쁘지 않을 것 같아서 적어봅니다..!